src/app/kvm/views/LXDClusterDetails/LXDClusterSummaryCard/LXDClusterSummaryCard.test.tsx

```diff
@@ -16,8 +12,7 @@ import {
   vmClusterState as vmClusterStateFactory,
   vmHost as vmHostFactory,
 } from "testing/factories";
-
-const mockStore = configureStore();
+import { renderWithBrowserRouter, screen, within } from "testing/utils";
```

Comment:
None of the tests in this file seem to require a router, you should be able to use `renderWithMockStore`

```suggestion
import { renderWithMockStore, screen, within } from "testing/utils";
```